src/backend/distributed/metadata/node_metadata.c

```diff
@@ -759,6 +759,15 @@ master_update_node(PG_FUNCTION_ARGS)
 			handle = StartLockAcquireHelperBackgroundWorker(MyProcPid, lock_cooldown);
 		}
 
+		if (!handle)
+		{
+			StringInfo lock_cooldown_string_value = makeStringInfo();
```

Comment:
Afaik we prefer camel case in code-base (unless we copy a function from postgres itself etc.)